src/payment/strategies/ppsdk/get-ppsdk-payment-id.ts

```diff
@@ -0,0 +1,10 @@
+import { CheckoutStore } from '../../../checkout';
+
+type GetPPSDKPaymentId = (store: CheckoutStore, methodId: string) => string | undefined;
+
+export const getPPSDKPaymentId: GetPPSDKPaymentId = (store, methodId) => {
```

Comment:
Largely habit/personal preference:

- No hoisting, only need to read code in one direction
- Singular `this` behaviour
- Option to have implicit returns
- Can't be accidentally used as a constructor
- No magic `arguments` array-like object value
- More semicolons 😛 

Nothing major at all, just removes a bit of possible magic/indirection

Reply:
I see. 🙂  For me, I like using named functions because their names are captured in stack traces, whereas for anonymous functions, they just show up as anonymous. I don't really have a problem with other problems listed like `arguments` and `this` because ESLint usually takes care of them. 🤷‍♂️

Reply:
Function name inference should cover the stack traces for us:
https://javascript.plainenglish.io/javascript-arrow-functions-how-to-debug-them-with-name-inference-ac0d7cd5c43b

We now have names for everything but the very anonymous IIFE:
![Screen Shot 2021-07-21 at 3 32 57 pm](https://user-images.githubusercontent.com/56807262/126440426-6d1e3477-2438-4707-a097-9f4a7ecacc49.png)

It _might_ be the case that certain minifier/uglifier setups could spoil this, I'm not 100% sure.

Reply:
Thanks @leeBigCommerce. I'm not too opinionated about it if you prefer to use arrow functions. One thing I do want to suggest is inline the type definition with the arrow function, i.e.: `const getPPSDKPaymentId = (store: CheckoutStore, methodId: string): string | undefined => {}`, considering you're not exporting the type and you don't have more than one implementation for the same type. But it's personal preference for me so 🍹